src/ray/gcs/format/gcs.fbs

```diff
@@ -108,7 +108,13 @@ table ResourcePair {
   value: double;
 }
 
+enum GcsTableNotificationMode:int {
+  APPEND_OR_ADD = 0,
+  REMOVE,
+}
+
 table GcsTableEntry {
+  mode: GcsTableNotificationMode;
```

Comment:
maybe rename `mode` to `notification_mode`, whose meaning is more clear.

Reply:
Thanks! Updated.